operate/qa/integration-tests/src/test/java/io/camunda/operate/it/ModifyProcessInstanceOperationZeebeIT.java

```diff
@@ -73,6 +74,7 @@ public void shouldCancelTokenForFlowNodeId() throws Exception {
   }
 
   @Test
+  @Ignore("operationIsFailed() check does not work correctly and times out")
```

Comment:
But what changed? The test seems to succeed in main branch. Or is it flaky? Should we research these failures?

Reply:
The issue is the predicate checks for operations require *all* operations have the same state. For example from the "operationsByProcessInstanceAreFailedCheck" check:

      return processInstance.getOperations().stream()
          .allMatch(
              operation -> {
                return operation.getState().equals(OperationState.FAILED);
              });
              
What happens is, say you have a test where you need to have a COMPLETED and a FAILED operation. This check will never pass. However we have a timeout feature, and when it times out it by default returns a success.

There's a few tests that have always been failing these checks but because the timeout threshold is reached, we continue on with the rest of the test. We would need these checks refactored to filter on the specific operation looking for failure/success.